HelpSource/Classes/ScopeView.schelp

```diff
@@ -74,54 +85,54 @@ EXAMPLES::
 SUBSECTION:: A step-by-step example
 
 code::
-// execute these in succession
-(
-s.boot;
-)
+s.boot; // boot server (if not already booted), then execute the following two blocks in succession:
+
 (
 f = Buffer.alloc(s,1024,2);
 b = Bus.audio(s,1);
 
 w=Window.new.front;
+w.onClose = { // free everything when finished
+  c.stop; a.free; d.free; f.free; b.free;
+  "SynthDefs, busses and buffers have all been freed.".postln;
+};
 c = ScopeView(w.view,w.view.bounds.insetAll(10,10,10,10));
 c.bufnum = f.bufnum;
+c.server_(s); // Important: one must assign the ScopeView to a server
```

Comment:
Writing this as `c.server = s;` would be more consistent with the rest of the style in this example.